### ISSUE TITLE
Error: cannot load such file -- cask/all

### ISSUE THREAD

**Issue by LeerGo** [buo/homebrew-cask-upgrade#169]
Hi all.

I encountered a problem when I run `brew cu`, i  can't solve it by myself. 

- this is the problem I encountered

```
> brew cu
Error: cannot load such file -- cask/all
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
/usr/local/Homebrew/Library/Taps/buo/homebrew-cask-upgrade/lib/bcu.rb:5:in `<top (required)>'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
/usr/local/Homebrew/Library/Taps/buo/homebrew-cask-upgrade/cmd/brew-cu.rb:67:in `<top (required)>'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
/usr/local/Homebrew/Library/Homebrew/utils.rb:83:in `require?'
/usr/local/Homebrew/Library/Homebrew/brew.rb:114:in `<main>'
```

- this is my environmental information, maybe help:

```
> brew  config

HOMEBREW_VERSION: 2.4.9-336-g264de96
ORIGIN: https://github.com/Homebrew/brew.git
HEAD: 264de96851258f6d478c819a11b973aee1ce0e3f
Last commit: 2 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: faf6322bbe6f76709abdfb14b487d02756b53add
Core tap last commit: 33 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: nvim
HOMEBREW_MAKE_JOBS: 12
http_proxy: http://127.0.0.1:1087
CPU: dodeca-core 64-bit kabylake
Homebrew Ruby: 2.6.3 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
Clang: 11.0 build 1103
Git: 2.28.0 => /usr/local/bin/git
Curl: 7.64.1 => /usr/bin/curl
Java: 14, 1.8.0_202, 1.6.0_65-b14-468
macOS: 10.15.6-x86_64
CLT: 1103.0.32.62
Xcode: 11.6
```

**Comment by LeerGo** [comment]
> Hi all.
> 
> I encountered a problem when I run `brew cu`, i can't solve it by myself.
> 
> * this is the problem I encountered
> 
> ```
> > brew cu
> Error: cannot load such file -- cask/all
> /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
> /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
> /usr/local/Homebrew/Library/Taps/buo/homebrew-cask-upgrade/lib/bcu.rb:5:in `<top (required)>'
> /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
> /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
> /usr/local/Homebrew/Library/Taps/buo/homebrew-cask-upgrade/cmd/brew-cu.rb:67:in `<top (required)>'
> /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
> /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
> /usr/local/Homebrew/Library/Homebrew/utils.rb:83:in `require?'
> /usr/local/Homebrew/Library/Homebrew/brew.rb:114:in `<main>'
> ```
> 
> * this is my environmental information, maybe help:
> 
> ```
> > brew  config
> 
> HOMEBREW_VERSION: 2.4.9-336-g264de96
> ORIGIN: https://github.com/Homebrew/brew.git
> HEAD: 264de96851258f6d478c819a11b973aee1ce0e3f
> Last commit: 2 hours ago
> Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
> Core tap HEAD: faf6322bbe6f76709abdfb14b487d02756b53add
> Core tap last commit: 33 minutes ago
> Core tap branch: master
> HOMEBREW_PREFIX: /usr/local
> HOMEBREW_CASK_OPTS: []
> HOMEBREW_EDITOR: nvim
> HOMEBREW_MAKE_JOBS: 12
> http_proxy: http://127.0.0.1:1087
> CPU: dodeca-core 64-bit kabylake
> Homebrew Ruby: 2.6.3 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
> Clang: 11.0 build 1103
> Git: 2.28.0 => /usr/local/bin/git
> Curl: 7.64.1 => /usr/bin/curl
> Java: 14, 1.8.0_202, 1.6.0_65-b14-468
> macOS: 10.15.6-x86_64
> CLT: 1103.0.32.62
> Xcode: 11.6
> ```

I have try #122  and #117  , but it's not wor for me.

**Comment by fangzhengjin** [comment]
same problem

**Comment by LeerGo** [comment]
it solved by pull request #168 

thx
